bundles/action/org.openhab.action.tinkerforge/META-INF/MANIFEST.MF

```diff
@@ -11,10 +11,12 @@ Bundle-ManifestVersion: 2
 Bundle-Description: This is the TinkerForge action of the open Home Aut
  omation Bus (openHAB)
 Import-Package: org.apache.commons.lang,
- org.openhab.core.scriptengine.action,
+ org.openhab.binding.tinkerforge.ecosystem,
+ org.openhab.binding.tinkerforge.internal.model,
```

Comment:
Actually, there shouldn't even a dependency from an action bundle to a binding bundle at all. The shared code should rather be externalized into an io bundle as it is e.g. done for squeezebox.